tests/karma/core/api/test_api.js

```diff
@@ -1,6 +1,6 @@
 import * as api from 'core/api';
 
-describe('search api', () => {
```

Comment:
These are pretty much all whitespace changes. Might be worth adding `w=1` to the URL for this file.